client/src/App.tsx

```diff
@@ -286,7 +288,11 @@ function App() {
     if (args.code) {
```

Comment:
The "code" argument is still supported so that old URLs continue to work.

Loading a URL like that will cause the URL to be immediately replaced in the address bar with a codez URL (if the code is compressible -- see below.)

Reply:
I will update the docs after I merged your PR; the `code` argument also has to keep working as is, because Zulip needs it for auto generating links from code blocks.

Reply:
Oh, that's cool!